packages/main/src/Table.js

```diff
@@ -484,6 +507,16 @@ class Table extends UI5Element {
 		}
 
 		ResizeHandler.register(this.getDomRef(), this._handleResize);
+
+		this._itemNavigation.setCurrentItem(this.rows.length ? this.rows[0] : this._columnHeader);
```

Comment:
Is this intended? I don't understand the full logic of the story, but it seems that every time the table is rerendered for any reason, the item navigation will be  reset to the first row.

Reply:
For some reason the column header has been included in the itemNavigation. Therefore, when the table receives focus for the first time via `TAB`, it gets focused instead of the first row. I am open to suggestions on better approches of applying this initial focus.